amr-wind/equation_systems/icns/icns_diffusion.H

```diff
@@ -77,6 +69,216 @@ struct DiffusionOp<ICNS, Scheme>
     }
 };
 
+class ICNSDiffScalarOp : public DiffSolverIface<amrex::MLABecLaplacian>
```

Comment:
Looking at the implementation of this class, it doesn't make sense for `ICNSDiffScalarOp` to be a subclass of `DiffSolverIface`. It is not using any of the data members and is override almost all methods. I recommend making this a standalone class and just having a clean implementation of `setup_operator`